task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +115,16 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return function () {
+        while (attempts) {
```

Comment:
this loop is going to be infinite if the function never returns  a value. Add an exit condition.